pkg/sac/resources/list.go

```diff
@@ -49,8 +49,9 @@ var (
 	// Risk.
 	DeploymentExtension = newResourceMetadata("DeploymentExtension", permissions.NamespaceScope)
 
-	Detection = newResourceMetadata("Detection", permissions.GlobalScope)
-	Image     = newResourceMetadata("Image", permissions.NamespaceScope)
+	Detection      = newResourceMetadata("Detection", permissions.GlobalScope)
+	Image          = newResourceMetadata("Image", permissions.NamespaceScope)
+	VirtualMachine = newResourceMetadata("VirtualMachine", permissions.NamespaceScope)
```

Comment:
I have next to know knowledge about how ocpvirt works so I will ask the following elementary question.  Does namespace apply?